src/main/java/seedu/address/model/pet/Sex.java

```diff
@@ -35,7 +35,13 @@ public static boolean isValidSex(String test) {
 
     @Override
     public String toString() {
-        return value;
+        if (value.equalsIgnoreCase("f")) {
+            return "Female";
+        } else if (value.equalsIgnoreCase("m")) {
+            return "Male";
+        } else {
+            return "Unknown";
```

Comment:
Test coverage does not cover this line. Since it should never occur, we should be throwing a RuntimeException?